src/hooks/useAudioPlayer.ts

```diff
@@ -301,7 +312,6 @@ export default function useAudioPlayer(
     (clipPath: string, previousClip: InternalClipPlayer, newConfig: InternalClipPlayer['config']): InternalClipPlayer => {
       const clip = { ...previousClip, config: newConfig };
       if (previousClip.config.preload !== newConfig.preload) {
-        clip.player.stop();
```

Comment:
The `stop` isn't required here. From the docs:

```
Unload and destroy a Howl object. This will immediately stop all sounds attached to this sound and remove it from the cache.
```